priv/schemas/v0-1-0/content-element.schema.json

```diff
@@ -886,6 +889,92 @@
         "$ref": "#"
       }
     },
+
+    "meaning-content": {
+      "anyOf": [
+        { "$ref": "#/$defs/media" },
+        { "$ref": "#/$defs/block" },
+        { "$ref": "#/$defs/text-block" }
+      ]
+    },
+
+    "definition-pronunciation": {
+      "properties": {
+        "type": {
+          "const": "pronunciation"
+        },
+        "src": {
+          "type": "string"
```

Comment:
The type here should be the same as the type for translation and meaning.  In other words, authors must be able to embed "rich text" into the pronunciation field.

```
<!ELEMENT %olcml.pronunciation.qname; (%olcml.inline.content;)*>
<!ATTLIST %olcml.pronunciation.qname;
  %olcml.core.attrs;
  %olcml.i18n.attrs;
  src			%URI;						#IMPLIED
  type			%ContentType;				#IMPLIED
>

<!ELEMENT %olcml.translation.qname; (%olcml.inline.content;)*>
<!ATTLIST %olcml.translation.qname;
  %olcml.core.attrs;
  %olcml.i18n.attrs;
>
```

Reply:
@darrensiegel Right now:

Meaning: accepts block elements
Translation & Pronunciation: accepts text blocks (and those can have inline rich text content in them)

The src attribute for pronunciation is for the audio file. I'm not a fan of the naming since that is confusing, but matched the name of src in the dtd.

Reply:
Ah yes! sorry, i saw `"type": "string"` and assumed that was the pronunciation content.